examples/00-basic-examples/Maxwell3DTeam3.py

```diff
@@ -0,0 +1,188 @@
+)
+M3D.modeler.primitives.create_cylinder(
+    cs_axis="Z", position=[0, "Coil_Position", 15], radius=20, height=20, name="Bore", matname="copper"
+)
+M3D.modeler.subtract("SearchCoil", "Bore", keepOriginals=False)
+M3D.modeler.section("SearchCoil", "YZ")
+M3D.modeler.separate_bodies("SearchCoil_Section1")
+M3D.modeler.primitives.delete("SearchCoil_Section1_Separate1")
+M3D.assign_current(["SearchCoil_Section1"], amplitude=1260, solid=False, name="SearchCoil_Excitation")
+
+################################################################################
+# Draw a line we will later use it to plot Bz on, (z-component of Flux Density)
+# A small diameter cylinder is also added to refine mesh locally around the line.
+Line_Points = [["0mm", "-55mm", "0.5mm"], ["0mm", "55mm", "0.5mm"]]
+P1 = primitives.create_polyline(Line_Points, name="Line_AB")
+P2 = primitives.create_polyline(Line_Points, name="Line_AB_MeshRefinement")
+P2.set_crosssection_properties(type="Circle", width="0.5mm")
+
+###############################################################################
+# Setup Maxwell 3D Model
+# ~~~~~~~~~~~~~~~~~~~~~~
+# Add analysis setup with frequency points at 50Hz and 200Hz
+Setup = M3D.create_setup(setupname="Setup1")
+Setup.props["Frequency"] = "200Hz"
+Setup.props["HasSweepSetup"] = True
+Setup.props["StartValue"] = "50Hz"
+Setup.props["StopValue"] = "200Hz"
+Setup.props["StepSize"] = "150Hz"
+Setup.update()
+
+################################################################################
+# Adjust Eddy Effects for LadderPlate and SearchCoil
+# NB, Eddy effect setting is ignored for stranded conductor type used in Search Coil
+M3D.eddy_effects_on(["LadderPlate"], activate=True)
+M3D.eddy_effects_on(["SearchCoil"], activate=False)
+
+################################################################################
+# Add a linear parametric sweep for the two coil positions
+sweepname = "CoilSweep"
+M3D.opti_parametric.add_parametric_setup("Coil_Position", "LIN -20mm 0mm 20mm", parametricname=sweepname)
+
+################################################################################
+# For now we need to use Optimetrics module to tick 'Save Fields' box
+Opt = M3D.ooptimetrics
+Opt.EditSetup(
+    "CoilSweep",
+    [
+        "NAME:CoilSweep",
+        "IsEnabled:=",
+        True,
+        ["NAME:ProdOptiSetupDataV2", "SaveFields:=", True, "CopyMesh:=", False, "SolveWithCopiedMeshOnly:=", True],
+        ["NAME:StartingPoint"],
+        "Sim. Setups:=",
+        ["Setup1"],
+        [
+            "NAME:Sweeps",
+            [
+                "NAME:SweepDefinition",
+                "Variable:=",
+                "Coil_Position",
+                "Data:=",
+                "LIN -20mm 0mm 20mm",
+                "OffsetF1:=",
+                False,
+                "Synchronize:=",
+                0,
+            ],
+        ],
+        ["NAME:Sweep Operations"],
+        ["NAME:Goals"],
+    ],
+)
```

Comment:
```suggestion
param = M3D.opti_parametric.add_parametric_setup("Coil_Position", 
                                                                                      "LIN -20mm 0mm 20mm", parametricname=sweepname)
param.props["ProdOptiSetupDataV2"]["SaveFields"] = True
param.props["ProdOptiSetupDataV2"]["CopyMesh"] = False
param.props["ProdOptiSetupDataV2"]["SolveWithCopiedMeshOnly"] = True
param.update()
```

Reply:
@DaveTwyman once you create an optimetrics setup in Pyaedt you get the object with properties so you can simply edit one of the props and update it once finished.

Reply:
@DaveTwyman of course in the example I've put all the three properties set, but if you need to change only SaveFields you can get rid of line 109 and 110

Reply:
> @DaveTwyman once you create an optimetrics setup in Pyaedt you get the object with properties so you can simply edit one of the props and update it once finished.

That's great, I do now recall seeing similar comment in the documentation. Let's leave the three lines in they are clear and adds detail to the example. Thanks